examples/threadcall_detect_in_image.jl

```diff
@@ -0,0 +1,56 @@
+using Images
+#Additional required packages to run this example.
+#Pkg.add("Images")
+
+using AprilTags
+
+# Simple method to show the image with the tags
+function showImage(image, tags)
+    # Convert image to RGB
+    imageCol = RGB.(image)
+    #traw color box on tag corners
+    foreach(tag->drawTagBox!(imageCol, tag), tags)
+    imageCol
+end
+
+
+##
+# Create default detector
+detector = AprilTagDetector()
+
+#Run against a file
+image = load(dirname(Base.source_path()) *"/../data/tagtest.jpg")
+
+@sync begin
+    starttime = Dates.value(now())
+    #call detector in thread
+    @async global t1 = @timed begin
+        println("time before detector $(Dates.value(now())-starttime) ms")
+        global tags = AprilTags.threadcalldetect(detector, image)
```

Comment:
`global` is slower since Julia optimizes at a per function level.  This is fine for testing, but try avoid for general implementation.